pyproject.toml

```diff
@@ -4,20 +4,24 @@ build-backend = "flit_core.buildapi"
 
 [project]
 name = "multiregex"
-authors = [{name="QuantCo, Inc.", email="noreply@quantco.com"}]
+authors = [
+  { name = "QuantCo, Inc.", email = "noreply@quantco.com" },
+  { name = "Jonas Haag", email = "jonas@lophus.org" },
+]
 
-maintainers = [{name = "Jonas Haag", email = "jonas.haag@quantco.com"}]
+maintainers = [{ name = "Bela Stoyan", email = "bela.stoyan@quantco.com" }]
 
 classifiers = [
-    "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.6",
-    "Programming Language :: Python :: 3.7",
-    "Programming Language :: Python :: 3.8",
-    "Programming Language :: Python :: 3.9",
-    "Programming Language :: Python :: 3.10",
-    "Programming Language :: Python :: 3.11"
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
 ]
 
+requires-python = ">=3.8"
+
+dependencies = ["pyahocorasick"]
 readme = "README.md"
 dynamic = ["version", "description"]
```

Comment:
Let's try to make this static (or use setuptools-scm where dynamic versions make sense)

Also see https://github.com/Quantco/polarify/blob/97fb1dadcbe17c69506a994f406720dad9fa07c4/polarify/__init__.py#L9-L14